lib/git_worktree.rb

```diff
@@ -320,7 +320,7 @@ def fetch_and_merge
 
   def fetch
     with_remote_options do |remote_options|
-      @repo.fetch(@remote_name, remote_options)
+      @repo.fetch(@remote_name, **remote_options)
```

Comment:
I don't think this one was you:  https://github.com/ManageIQ/manageiq/commit/bdb307e2c516ea549abbb99802163b6d67efac23